packages/core/admin/admin/src/translations/de.json

```
@@ -26,21 +30,31 @@
   "Auth.form.error.user.not-exist": "Diese E-Mail-Adresse ist nicht registriert.",
   "Auth.form.error.username.taken": "Dieser Benutzername ist bereits vergeben",
   "Auth.form.firstname.label": "Vorname",
-  "Auth.form.firstname.placeholder": "Max",
+  "Auth.form.firstname.placeholder": "z.B. Max",
   "Auth.form.forgot-password.email.label": "Gib deine E-Mail ein",
   "Auth.form.forgot-password.email.label.success": "Eine E-Mail wurde erfolgreich verschickt an",
   "Auth.form.lastname.label": "Nachname",
-  "Auth.form.lastname.placeholder": "Mustermann",
+  "Auth.form.lastname.placeholder": "z.B. Mustermann",
+  "Auth.form.password.hide-password": "Passwort ausblenden",
+  "Auth.form.password.hint": "Das Passwort muss mindestens 8 Zeichen, 1 Großbuchstabe, 1 Kleinbuchstabe und 1 Zahl enthalten",
```

What do you think about?

```suggestion
  "Auth.form.password.hint": "Das Passwort muss mindestens 8 Zeichen, einen Großbuchstaben, einen Kleinbuchstaben und eine Zahl enthalten",
```

---

Seems more common like that, I'll change that